stdlib/testing/testing.flux

```diff
@@ -9,7 +9,17 @@ builtin diff
 loadStorage = (csv) => c.from(csv: csv)
 loadMem = (csv) => c.from(csv: csv)
 
-test = (name, input, want, testFn) => {
-    got = input |> testFn()
-    return got |> diff(want: want) |> yield(name: name) |> assertEmpty()
+run = (case) => {
+    tc = case()
+    got = tc.input |> tc.fn()
+    dif = got |> diff(want: tc.want)
+    pass = dif |> yield(name: "diff") |> assertEmpty()
```

Comment:
do we need to yield here, if we're also returning it in the result?

Reply:
I think we do need it because if assertEmpty fails(outside of the repl) then yield is the only way to get to the diff.